### PR TITLE
Navigation: Prevent chevron overlaying navbar menus

### LINKED ISSUE #47946: Expandable Sidemenu: portalled hover menu overlays chevron

### PR DESCRIPTION
**Pull request by ashharrison90**
**What this PR does / why we need it**:

| before | after |
| --- | --- |
| image | image |

**Which issue(s) this PR fixes**:

Fixes #47946 

**Special notes for your reviewer**:

### CODE REVIEW

File: public/app/core/components/NavBar/Next/NavBarNext.tsx [hunk 1]
```diff
@@ -75,6 +82,8 @@ export const NavBarNext = React.memo(() => {
           }}
         >
           <FocusScope>
+            <div id={NAV_MENU_PORTAL_CONTAINER_ID} className={styles.menuPortalContainer} />
```
Comment (axelavargas) [inline public/app/core/components/NavBar/Next/NavBarNext.tsx:85]: Nit: I think would be nice if we are more explicit here, maybe we can create a new component called `<NavPortalContainer>` the same approach we have in `AppWrapper`?

**Review by axelavargas — APPROVED** [review]
Mr Harrison, code looks good to me and works as expected, only a minor nit 🙈

**Comment by JoaoSilvaGrafana** [comment]
I have found a slight difference from the previous behaviour, this is what happens when you are in a sub menu and press Shift+Tab currently:

https://user-images.githubusercontent.com/100691367/164505327-ee0939d7-4239-4d88-a961-3f9b4bd3b5ab.mov

It is not perfect because it actually scrolls the page to the bottom (maybe a papercut thing to add?), but it does keep you in the nav bar like it did previously.

With this change and the addition of that `menuPortalContainer`, pressing Shift+Tab from a sub menu will focus on the address bar and then if you tab again it will have a strange focus around the whole sub menu:

https://user-images.githubusercontent.com/100691367/164506293-696418ae-25d8-4cf0-9359-6169edc8926c.mov

I guess we should decide if we want to fix in this branch or have another ticket to fix that behaviour.

**Comment by ashharrison90** [comment]
@JoaoSilvaGrafana mmmm, that's annoying

i'll look into it tomorrow morning 😄

**Comment by ashharrison90** [comment]
nm, fixed it 🥳

**Review by JoaoSilvaGrafana — APPROVED** [review]
Great job! Even fixed the other bug with the Shift+Tab I mentioned 🎉

**Pull request merged**